Tests/XCUITests/BookmarkingTests.swift

```diff
@@ -130,7 +130,9 @@ class BookmarkingTests: BaseTestCase {
     }
 
     func testBookmarksAwesomeBar() {
-        waitForExistence(app.buttons["urlBar-cancel"], timeout: 15)
+        XCTExpectFailure("The app was not launched", strict: false) {
```

Comment:
Such a code block could belong in `setUp()`. I'm not sure if we should see if the app has been launched from `BaseTestCase`.

Reply:
Not sure either, with this code we are skiping the test if it fails...